modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -841,6 +841,34 @@ square grouping and ordering algorithm fails.
 CV_EXPORTS_W bool findChessboardCorners( InputArray image, Size patternSize, OutputArray corners,
                                          int flags = CALIB_CB_ADAPTIVE_THRESH + CALIB_CB_NORMALIZE_IMAGE );
 
+/** @brief Finds the positions of internal corners of the chessboard using a sector based approach.
+
+@param image Source chessboard view. It must be an 8-bit grayscale or color image.
+@param patternSize Number of inner corners per a chessboard row and column
+( patternSize = cvSize(points_per_row,points_per_colum) = cvSize(columns,rows) ).
+@param corners Output array of detected corners.
+@param flags operation flags for future improvements
+
+The function is analog to findchessboardCorners but uses a localized radon
+transformation approximated by box filters being more robust to all sort of
+noise, faster on larger images and is able to directly return the sub-pixel
+position of the internal chessboard corners. The Method is based on the paper
```

Comment:
I think you should add the bibtex reference of the paper [here](https://github.com/opencv/opencv/blob/master/doc/opencv.bib).

Then you can do:

> The method is based on the paper @cite Duda2018 "Accurate Detection and Localization of Checkerboard Corners for Calibration" by A. Duda and U. Frese, BMVC 2018...

`@cite` will add a link to the bibliography in the form `[20]` but not the text.